test/katex-spec.js

```diff
@@ -2950,6 +2950,15 @@ describe("Unicode", function() {
     it("should parse binary operators", function() {
         expect("±×÷∓∔∧∨∩∪≀⊎⊓⊔⊕⊖⊗⊘⊙⊚⊛⊝⊞⊟⊠⊡⊺⊻⊼⋇⋉⋊⋋⋌⋎⋏⋒⋓⩞\u22C5").toParse();
     });
+
+    it("should parse delimeters", function() {
+        expect("\\left\u230A\\frac{a}{b}\\right\u230B").toParse();
+        expect("\\left\u2308\\frac{a}{b}\\right\u2308").toParse();
+        expect("\\left\u27ee\\frac{a}{b}\\right\u27ef").toParse();
+        expect("\\left\u27e8\\frac{a}{b}\\right\u27e9").toParse();
+        expect("\\left\u23b0\\frac{a}{b}\\right\u23b1").toParse();
+        expect("┌x┐ └x┘").toParse();
+    });
```

Comment:
Can you change all of these to `toBuild`? I'm pretty sure only then will your new code be tested, as it's run only during build time, not parse time.  `toBuild` also checks `toParse`, so it's strictly stronger.